qa.gemspec

```diff
@@ -22,10 +22,13 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.6"
   s.add_dependency "activerecord-import"
   s.add_dependency "deprecation"
+  s.add_dependency "rdf", "2.2.3"
```

Comment:
This seems like a problem; my sufia 7.3 app currently has 2.2.5 which means I won't be able to upgrade to the next release of qa